component/libfuse/libfuse_handler.go

```diff
@@ -241,6 +241,12 @@ func (lf *Libfuse) destroyFuse() error {
 
 //export libfuse_init
 func libfuse_init(conn *C.fuse_conn_info_t, cfg *C.fuse_config_t) (res unsafe.Pointer) {
+	// fuse started processing requests, no error happened during mounting
+	log.Trace("Libfuse::libfuse_init : notifying parent")
+	ppid := syscall.Getppid()
```

Comment:
This call will not come back to blobfuse if libfuse decided to fail to init and in such case parent process will go in wait forever.

Reply:
I think in that case child would exit and cause parent to exit.

Reply:
but parent is waiting for SIGUSR2 and as child exited parent will not receive that?

Reply:
parent will receive SIGCHILD then and exit. Related code is in [here](https://github.com/Azure/azure-storage-fuse/pull/1088/files#diff-57454978dcd397e797f4c48e4b1f80b4b249655d5fc975ad55c54d560ba01de1R446-R448).